### PR TITLE
Doc and cleanup

### PR DESCRIPTION
**Pull request by olinox14**
some cleanup and new readme

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+We made the following choices so far:
+
+#### No JS  
+
+Website shall be accessible via TOR or by users who disabled JS on their browser
+
+#### No framework
+
+Time is a challenge here, no time to learn somtehing common accross all of the volunteers.
```
Comment (antarctic-cheetah) [inline readme.md]:
```suggestion
Time is a challenge here, no time to learn something common across all of the volunteers.
```

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+Website has to be designed mobile first, since it is the most used terminal used by the one who 
+need it.
+
+## Setup the project
+
+First, install python 3.8+: <https://www.python.org/downloads/release/python-3910/>
+
+Then run in the project directory:
+
+    pip install -r requirement.txt
```
Comment (antarctic-cheetah) [inline readme.md]:
```suggestion
    pip install -r requirements.txt
```

File: templates/home.html [hunk 3]
```diff
@@ -4,11 +4,13 @@
...
-        <h2>Welcome</h2>
+        <h2>Ukrainianconflict-live</h2>
 
         <p>
-            This is a collaboration between volunteers organized from /r/UkrainianConflict to create a single hub of information regarding the ongoing situation.
-            This is a work in progress, so if you'd like to make any suggestions to improve the site, please submit a comment in the dedicated <a href="#">Reddit thread</a>.
+            This is a initiative from /r/UkrainianConflict volunteers to create a single hub of information
+            regarding the ongoing situation. This is a work in progress.
+            If you'd like to make any suggestions to improve the site,
+            <a href="https://github.com/Ukrainianconflict-live/website">make us know</a>.
```
Comment (antarctic-cheetah) [inline templates/home.html]:
```suggestion
            <a href="https://github.com/Ukrainianconflict-live/website">let us know</a>.
```

File: readme.md (hunk 2 above)
Comment (olinox14) [reply readme.md]: thanks, god I was really tired yesterday night to make such mistakes. I'll fix it soon

**Pull request merged**